frontend/src/static/js/v_summary.js

```diff
@@ -156,6 +167,17 @@ window.vSummary = {
 
       return totalLines / totalCount;
     },
+
+    allGroupsMerged: {
+      get() {
+        return this.isAllGroupsMerged();
```

Comment:
is it necessary to separate into two (allGroupsMerged and isAllGroupsMerged)? seems like the is hard to distinguish the two of them based on their names, which may confuse developers in the future

Reply:
Do you know if there is anyway I can get a computed variable (`allGroupsMerged`) from a method? If there is a way then I don't need the `isAllGroupsMerged` method anymore.

Reply:
`this.allGroupsMerged`?

Reply:
Oh okay it works. I thought I tried that before but somehow it didn't work. Maybe I mistakenly added brackets after.